src/rpdk/core/contract/resource_client.py

```diff
@@ -201,6 +215,78 @@ def get_metadata(self):
                 and properties[prop]["insertionOrder"] == "false"
             }
 
+    def transform_model(self, input_model, output_model):
+        if self.property_transform_keys:
+            self.check_npm()
+        for prop in self.property_transform_keys:
+            document_input, _path_input, _parent_input = traverse(
+                input_model, list(prop)[1:]
+            )
+            document_output, _path_output, _parent_output = traverse(
+                output_model, list(prop)[1:]
+            )
+            if document_input != document_output:
+                transformed_property = self.transform(prop, input_model)
+                self.update_transformed_property(
+                    prop, transformed_property, input_model
+                )
+
+    def transform(self, property_path, input_model):
+
+        path = "/" + "/".join(property_path)
+        property_transform_value = self.property_transform[path].replace('"', '\\"')
```

Comment:
```suggestion
        property_transform_value = json.dumps(self.property_transform[path])
```

Might help to catch [other escaped characters](https://stackoverflow.com/questions/18886596/replace-all-quotes-in-a-string-with-escaped-quotes)